### PR TITLE
Add ddSpan to RequestContext by default

### PR DESCRIPTION
**Pull request by klesgidis**

### CODE REVIEW

**Review by nikostoulas — COMMENTED** [review]
@klesgidis Note that we have added a middleware to add route names in ddSpan. See https://github.com/Workable/orka/blob/ab78fe6cde4f17888ead71ff864d67f6c12be77b/src/middlewares/datadog-match-routes.ts#L3
Are you using ddSpan somewhere else?

**Comment by klesgidis** [comment]
> @klesgidis Note that we have added a middleware to add route names in ddSpan. See
> 
> https://github.com/Workable/orka/blob/ab78fe6cde4f17888ead71ff864d67f6c12be77b/src/middlewares/datadog-match-routes.ts#L3
> 
> 
> Are you using ddSpan somewhere else?

I am using it on our `Metrics` module in careers-page, but I believe it is useful to exist on the requestcontext. WDYT?

**Comment by nikostoulas** [comment]
> > @klesgidis Note that we have added a middleware to add route names in ddSpan. See
> > https://github.com/Workable/orka/blob/ab78fe6cde4f17888ead71ff864d67f6c12be77b/src/middlewares/datadog-match-routes.ts#L3
> > 
> > Are you using ddSpan somewhere else?
> 
> I am using it on our `Metrics` module in careers-page, but I believe it is useful to exist on the requestcontext. WDYT?

Sure, can you check the failing tests?

File: src/initializers/koa/add-request-context.ts [hunk 1]
```diff
@@ -8,6 +8,11 @@ export default function(als: AsyncLocalStorage<Map<string, any>>): Middleware {
       if (ctx.state.requestId) {
         store.set('requestId', ctx.state.requestId);
       }
+      // @ts-ignore
```
Comment (nikostoulas) [inline src/initializers/koa/add-request-context.ts:11]: ❓ what does ts-ignore do?

**Review by nikostoulas — APPROVED** [review]

**Pull request merged**